packages/file-explorer/src/FileList.tsx

```diff
@@ -485,6 +488,18 @@ export const FileList = (props: FileListProps): JSX.Element => {
     }
   }, [draggedItems, dropTargetItem]);
 
+  const { isNewItemModalClicked, focusedPath } = props;
+  useEffect(() => {
+    if (isNewItemModalClicked && focusedPath !== undefined) {
```

Comment:
Is this needed? `isNewItemModalClicked` was always false, so I'm not sure this code ever ran.